ui/app/AppLayouts/Communities/panels/SharedAddressesAccountSelector.qml

```diff
@@ -315,9 +315,7 @@ StatusListView {
                 ButtonGroup.group: d.airdropGroup
                 anchors.verticalCenter: parent.verticalCenter
                 icon.name: "airdrop"
-                icon.color: hovered ? Theme.palette.primaryColor3 :
-                                      checked ? Theme.palette.primaryColor1
-                                              : disabledTextColor
+                icon.color: checked ? Theme.palette.primaryColor1 : disabledTextColor
```

Comment:
is `hovered` removed by design for this button?

Reply:
yeah, I checked against Figma again, it looked really odd here; we just need to change the background color, not the icon color here